.github/workflows/publish_pypi.yml

```
@@ -249,7 +279,7 @@ jobs:
           repository-url: https://test.pypi.org/legacy/
 
   open_failure_issue:
-    needs: [build_windows_wheels, build_macos_and_linux_wheels, build_sdist]
+    needs: [build_windows_wheels, build_macos_and_linux_wheels, build_macos_arm64_wheels, build_sdist]
```

This could have made our wheel builds fail silently, just caught it

---

I resolved this in #4015 and merged `develop` to get this here.